cluster.ml

```diff
@@ -70,6 +70,7 @@ let deployment =
       ~zone:gcloud_zone
       ~java:`Oracle_7
       ~machine_type:(`Google_cloud `Highmem_8)
+      ~additional_packages:["postgresql-9.4"]
```

Comment:
Hmm good point; we might want to add some sort of ~on_boot_scripts options. If this is okay for now, though, maybe just go with it & I can file an issue for it? I'm up for either.